srcpkgs/arpack-ng/template

```diff
@@ -1,8 +1,9 @@
 # Template file for 'arpack-ng'
 pkgname=arpack-ng
-version=3.7.0
+version=3.8.0
 revision=1
 build_style=gnu-configure
+configure_args="--enable-static --enable-shared"
```

Comment:
I think you need to drop enable-shared from here?

Reply:
Oops, fixed. What is the general policy w.r.t. static libraries? Should they be included if at all possible?

Reply:
If it's easy to build them, I'm okay with including. This doesn't much look like something that would be used in a static binary that needs to be used in multiple places, so imo it's ok to omit it.